core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -52,22 +52,20 @@ $.extend( $.fn.dataTable.ext.type.order, {
 function stageEndPoint(appId) {
     var urlRegex = /https\:\/\/[^\/]+\/([^\/]+)\/([^\/]+)\/([^\/]+)?\/?([^\/]+)?\/?([^\/]+)?\/?([^\/]+)?/gm;
     var urlArray = urlRegex.exec(document.baseURI);
-    var ind = urlArray.indexOf("proxy");
+    var indexOfProxy = urlArray.indexOf("proxy");
```

Comment:
we seems to be using urlArray and then words, we don't really need both.  Just get everything form words if that is what you need the format in.

Reply:
Done